### DISCUSSION TITLE
Rename forge-pull to forge-pull-topics?

### DISCUSSION THREAD

**Discussion by DivineDominion** [magit/forge#495] [Ideas]
It's maybe just a personal quirk, but I went with `forge-add-repository` and then `N f f` -- until I just now revisited the manual and realized that `forge-pull` is basically just that. Out of caution I have 'skipped' over `forge-pull` in the past. I think I was sub-consciously mistaking this for a git pull which I only cautiously perform in team projects.

So yeah, this is just an anecdotal data point that `forge-pull` misled me a bit.  I'd personally like `forge-pull-topics` better -- I know nobody asked for this opinion though ;)

**Comment by tarsius** [comment]
Yeah... I am going to stick to the current name. :grinning:
